src/vcpkg/base/json.cpp

```diff
@@ -253,7 +253,7 @@ namespace vcpkg::Json
     }
     Value Value::number(double d) noexcept
     {
-        vcpkg::Checks::check_exit(VCPKG_LINE_INFO, isfinite(d));
+        vcpkg::Checks::check_exit(VCPKG_LINE_INFO, std::isfinite(d));
```

Comment:
This one was intentional paranoia due to `isfinite` possibly being a macro. We use `math.h` not `cmath` so we can only depend on the global namespace name, not the `std::` namespace name.